tests/testbase.py

```diff
@@ -18,11 +17,11 @@ def setUp(self) -> None:
         PersistentObjectStorage().storage_file = self.response_file
         PersistentObjectStorage().dump_after_store = True
         PersistentObjectStorage()._is_write_mode = True
-        StoreFiles.counter = 0
 
     def tearDown(self) -> None:
         pass
         super().tearDown()
+        return None
```

Comment:
?

Reply:
just when I were debugging it, and want to escape tearDown earlier without comment out the lines